<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>951</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>951</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Docente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>931</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Registro:</string>
    </property>
    <widget class="QTableWidget" name="tblDocente">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>901</width>
       <height>351</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>351</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>11111</width>
       <height>351</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID_Docente</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID_Persona</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ape_Paterno</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ape_Materno</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom1</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom2</string>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DNI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Departamento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Provincia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Distrito</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Direccion</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Celular</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Correo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fec_Nacimiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sexo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Especialidad</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>931</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Datos:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_Docente:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIdDocente">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Paterno:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtApepaterno">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Materno:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtApematerno">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNom1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Primer Nombre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Segundo Nombre:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNom2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>DNI:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDNI">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Distrito:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDistrito">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtProvincia">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Departamento:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Dirección:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Provincia:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDireccion">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDepartamento">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Celular:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCelular">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Correo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCorreo">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nacimiento:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFecnacimiento">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sexo:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbMasculino">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>M</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdFemenino">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnActualizar">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegistrar">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLimpiar">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>240</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEspecialidad">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Especialidad:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_Persona:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIdPersona">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
